sass/utilities/mixins.sass

```diff
@@ -173,8 +173,10 @@
 // Responsiveness
 
 $tablet: 769px !default
-$desktop: 1000px !default // 960px container + 40px
-$widescreen: 1192px !default // 1152px container + 40
+// 960px container + 40px
+$desktop: 1000px !default
+// 1152px container + 40
+$widescreen: 1192px !default
```

Comment:
Googling this very issue I'm finally arriving here. I would greatly appreciate @jgthms merging this PR.